### PR TITLE
Update content and design on WAF error page.

### PR DESCRIPTION
**Pull request by tgandrews**
### What is changing
Change to match new design and using the CDN. The CDN is using https://sdc-global-design-patterns.netlify.com/

But still has a couple of outstanding problems:
- [x] Google analytics UA is missing
- [x] How are we getting the browser and OS version?
- [x] What browsers are we supporting?
- [x] Should we check print as well as in browser?

### How to review
1. Ensure this matches the design

### CODE REVIEW

**Comment by tgandrews** [comment]
[Update WAF Error HTML to Fix it](https://trello.com/c/1grEBx0U/937-update-waf-error-html-to-fix-it)

**Review by samiwel — APPROVED** [review]
Content looks correct as per the design attached to the card. @ajmaddaford what's the process for updating this error page on the WAF?

File: waf_error_page.html [hunk 1]
```diff
@@ -1,145 +1,165 @@
...
+        "script",
+        "https://www.google-analytics.com/analytics.js",
+        "ga"
+      );
+
+      ga("create", "", "auto");
+      ga("set", "anonymizeIp", true);
+      ga("send", "pageview");
+    </script>
+    <!-- End Google Analytics -->
```
Comment (ajmaddaford) [inline waf_error_page.html]: I think we should remove the Google Analytics code. The WAF error page is for all Survey Data Collection products and there is no combined Google Analytics account we could use here.

File: waf_error_page.html [hunk 2]
```diff
@@ -1,145 +1,165 @@
...
+    <script>
+      window.__EQ_SESSION_TIMEOUT__ = null;
+      window.__EQ_SESSION_TIMEOUT_PROMPT__ = null;
+      window.__EQ_SESSION_CONTINUE_URL__ = "/timeout-continue";
+      window.__EQ_SESSION_EXPIRED_URL__ = "/session-expired";
+    </script>
```
Comment (ajmaddaford) [inline waf_error_page.html]: These timeout values aren't needed.

File: waf_error_page.html [hunk 3]
```diff
@@ -1,145 +1,165 @@
...
+        <div class="page__subheader">
+          <div class="container"></div>
+        </div>
+        <div class="page__container container">
+          <div class="grid grid--reverse">
+            <div class="grid__col col-4@m"></div>
+
+            <div class="grid__col col-7@m pull-1@m">
+              <main role="main" id="main" class="page__main">
+                <h1 class="saturn u-fs-xl">An error has occurred</h1>
```
Comment (ajmaddaford) [inline waf_error_page.html]: I thought all the planet based classes were removed?

File: waf_error_page.html [hunk 4]
```diff
@@ -1,145 +1,165 @@
...
+              </main>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <footer class="page__footer">
+        <div class="footer page__footer" role="contentinfo">
+          <div class="container">
+            <div class="pluto">
```
Comment (ajmaddaford) [inline waf_error_page.html]: Same as comment above about planet based classes.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
@tgandrews I'm waiting to hear back about whether the `{{ERROR-CODE}}` and `{{ERROR-ID}}` are used, I'll let you know as soon as I hear back.

File: waf_error_page.html [hunk 5]
```diff
@@ -1,145 +1,129 @@
...
+                    <a
+                      href="tel:03001234931"
+                      aria-describedby="survey-help"
+                      >0300 1234 931</a>
+                  </p>
+                </div>
+
+                <div class="u-mt-m u-mb-s">
+                  <div class="u-fw-b">System information</div>
+                  <div>{{ ERROR - CODE }} {{ ERROR - ID }}</div>
```
Comment (ajmaddaford) [inline waf_error_page.html]: These need to be `{{ERROR-CODE}}` and `{{ERROR-ID}}` - without spaces.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**